From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF: (re)raise the exception always unless returning
  otherwise leads atm to masking of this error while testing on i386 and then
failling since

UnboundLocalError: local variable unser referenced before assignment

More detail:
https://buildd.debian.org/status/fetch.php?pkg=pandas&arch=i386&ver=0.19.1-1&stamp=1479504883

--- a/pandas/io/tests/json/test_pandas.py
+++ b/pandas/io/tests/json/test_pandas.py
@@ -167,7 +167,7 @@ class TestPandasContainer(tm.TestCase):
                 if raise_ok is not None:
                     if isinstance(detail, raise_ok):
                         return
-                    raise
+                raise
 
             if sort is not None and sort in unser.columns:
                 unser = unser.sort_values(sort)
